tests/test_migration.py

```diff
@@ -0,0 +1,148 @@
+import random
```

Comment:
could we rename this file `test_migrate_to_single_table`? and if possible, move it to `postgres/migrations`?

Reply:
I changed the name of the file but moving it to the `postgres/migrations` directory makes the second test get stuck at migration for some reason